Misc/NEWS.d/next/Library/2024-10-03-17-13-22.gh-issue-124944.YyLAzf.rst

```diff
@@ -0,0 +1 @@
+Add ``SO_ORIGINAL_DST`` to the ``sockets`` module.
```

Comment:
Please use the `:mod:` label reference module.
```
Add :attr:`SO_ORIGINAL_DST` to the :mod:`sockets`.
```
For more information, to see https://devguide.python.org/documentation/markup/